Tools/TestWebKitAPI/Tests/mac/WKWebViewMacEditingTests.mm

```diff
@@ -193,6 +273,40 @@ - (NSRange)range
     EXPECT_WK_STREQ(textContent, [webView stringByEvaluatingJavaScript:@"document.body.textContent"]);
 }
 
+TEST(WKWebViewMacEditingTests, KeyDownFiresBeforeCompositionEvent)
+{
+    __block bool isDone = false;
+
+    auto configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
```

Comment:
I _think_ we prefer `RetainPtr` over `auto` per smart pointer adoption guidelines?

```suggestion
    RetainPtr configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
```

Reply:
Fixed.